app/views/layouts/application.html.erb

```diff
@@ -6,15 +6,24 @@
 
     <%= stylesheet_link_tag    'application', media: 'all' %>
     <%= javascript_include_tag 'application' %>
+    <style>
```

Comment:
This should live in `application.css`

Reply:
Sounds fine to me, those lines are from the base Turing repo and was not added by us.